python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -144,7 +145,12 @@ def measure(self, measRecord, exposure):
         xmy2 = xmy*xmy
         xy2 = Ixy*Ixy
         a2 = 0.5 * (xpy + np.sqrt(xmy2 + 4.0*xy2))
-        sigma = exposure.getPsf().getSigma()
+
+        # Get the width of the PSF at the center of the trail
+        center = Point2D(xc, yc)
+        sigma = exposure.getPsf().computeShape(center).getTraceRadius()
```

Comment:
A question that is more of scientific in nature and less of technical: do you want the determinant radius from `getDeterminantRadius` or trace radius? They agree for a circular Gaussian, but in general they differ. I tried to look at what needs to be passed in `findLength` in L155, and it doesn't seem to agree with the docstring. I don't know the algorithm well enough to know if that's an error, but I wanted to bring it up.